apps/docs/app/(docs)/[[...slug]]/page.tsx

```
@@ -25,6 +24,37 @@ export default async function Page(props: {
 
   const MDX = page.data.body
 
+  const netlifyLogo = (
+    <div className="w-24 dark:text-white text-black">
+      <svg
+        viewBox="0 0 512 209"
+        xmlns="http://www.w3.org/2000/svg"
+        className="fill-current" // Add fill-current here for Tailwind
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)